bibliopixel/led/coord_map.py

```diff
@@ -0,0 +1,143 @@
+from .. util import pointOnCircle
+
+
+class Rotation:

```

Comment:
coord_map is dead. Also, Rotation used to be called MatrixRotation but I use it for more than matrices now.